canonical-web-and-design/vanilla-framework
# Code snippet: multiple dropdowns

## Done

- Added ability to support multiple dropdowns in `p-code-snippet__header`
- Alternative layout available by adding the `is-stacked` utility to `p-code-snippet__header`

Fixes #3470 

## QA

- Open [demo](https://vanilla-framework-3527.demos.haus/docs/examples/patterns/code-snippet/dropdown-multiple)
- See that the dropdowns behave as expected, and match the [design](https://app.zeplin.io/project/5fa270d8eee4dbb267e579f1/screen/5ffc2fdcb65e6a99b78f9d47)
- Review updated documentation:
  - [Docs](https://vanilla-framework-3527.demos.haus/docs/base/code#code-snippet)
  - [Component status](https://vanilla-framework-3527.demos.haus/docs/component-status)
